src/analytics-library/extensions/spring/get_collection/http-response.md

```diff
@@ -0,0 +1,9 @@
+*Response 200*
+
+``` json
+[ {
+  "scope" : "travel-sample/inventory",
+  "hash_md5" : "<MD5 hash>",
```

Comment:
Please check this response and add a real example MD5 hash, if it's not too long

Reply:
Cool, added.